lib/protoboeuf/google.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+# There isn't a clean 1:1 mapping between constants and *.rb files, so eager load instead of autoload.
+
+Dir[File.expand_path("**/*.rb", __dir__)].each { |file| require file }
```

Comment:
This is catching too much, we should limit it to files below the "google" dir
```suggestion
Dir[File.expand_path("google/**/*.rb", __dir__)].each { |file| require file }
```